modules/products/bamboo/main.tf

```diff
@@ -30,12 +30,13 @@ module "database" {
 module "nfs" {
   source = "../../AWS/nfs"
 
-  namespace         = var.namespace
-  product           = local.product_name
-  requests_cpu      = var.nfs_requests_cpu
-  requests_memory   = var.nfs_requests_memory
-  limits_cpu        = var.nfs_limits_cpu
-  limits_memory     = var.nfs_limits_memory
-  availability_zone = var.eks.availability_zone
-  shared_home_size  = var.shared_home_size
+  namespace            = var.namespace
+  product              = local.product_name
+  requests_cpu         = var.nfs_requests_cpu
+  requests_memory      = var.nfs_requests_memory
+  limits_cpu           = var.nfs_limits_cpu
+  limits_memory        = var.nfs_limits_memory
+  availability_zone    = var.eks.availability_zone
+  shared_home_size     = var.shared_home_size
+  cluster_service_ipv4 = "172.20.2.2"
```

Comment:
lower band was picked to avoid collision
This is based on K8S IP address distribution algorithm
https://kubernetes.io/blog/2022/05/23/service-ip-dynamic-and-static-allocation/#avoid-ClusterIP-conflict

Reply:
again, please use local variables rather than the constant literals.